### DISCUSSION TITLE
Livewire save form inputs without wire:model

### DISCUSSION THREAD

**Discussion by aspyropoulos** [livewire/livewire#3325] [1. Help]
I am using tippyjs to show a modal but that's little problematic as LW cannot update the input fields inside there.
What I want to do is to save my form without using `wire:model` ie. 

```
<form wire:submit.prevent="save">
  <input type"text" name="sizes[]">
  <input type"text" name="sizes[]">
 <button type="submit">Save</button>
</form>
```

and then get the values at POST

```
function save() {
  $sizes = $request->sizes; // this is not working
}
```
How can I do this?

**Comment by aspyropoulos** [comment]
I end up using pure javascript to and make calls using global livewire javascript object.

**Comment by ph7jack** [comment]
why you dont use a modal component compatible with livewire?

**Reply by aspyropoulos** [reply]
Well the form that I want to parse is inside tippyjs (meaning that part of the dom is been consumed by that plugin). Thus, LW cannot use that same part also properly update/values etc.
